css/main.scss

```diff
@@ -371,6 +371,13 @@ i:hover {
   }
 }
 
+.header {
+  background-color: #fff;
+  padding: 10px 0;
+  font-size: 12px;
+  text-align: center;
+}
```

Comment:
We should make the content right-aligned as it is in the footer. You'll probably have to move the content into a `<div class="col-xs-12 ">` for that.